src/notebooks/controllers/vscodeNotebookController.ts

```diff
@@ -580,26 +579,8 @@ export class VSCodeNotebookController implements Disposable, IVSCodeNotebookCont
             traceInfo('Switch kernel did not change kernel.');
             return;
         }
-        switch (this.connection.kind) {
```

Comment:
Old code here was to distinguish between existing kernels and "interpreters as kernels" the other code path doesn't exist anymore, and the language info here it capture in the Telemetry.SwitchKernel event below.